package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "oppia",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "dependencies": {
```

Comment:
This should probably be devDependencies.

Reply:
updated.